### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Upgrade to v.2.0.2

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/wetty_ynh%20PR1%20(ericgaspar)/)

### CODE REVIEW

File: scripts/upgrade [hunk 1]
```diff
@@ -114,9 +96,9 @@ fi
 #=================================================
 # NGINX CONFIGURATION
 #=================================================
-ynh_script_progression --message="Upgrading nginx web server configuration..." --weight=1
+ynh_script_progression --message="Upgrading NGINX web server configuration..." --weight=1
 
-# Create a dedicated nginx config
+# Create a dedicated NGIN config
```
Comment (yalh76) [inline scripts/upgrade:101]:
```suggestion
# Create a dedicated NGINX config
```

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Review by yalh76 — APPROVED** [review]

**Comment by Salamandar** [comment]
I'm sorry I have a lot on my plate these times. It'll be quieter in 2 weeks. Sorry for the wait…

**Comment by ericgaspar** [comment]
No problem. Hope you don't mind me pushing a bunch of commits

**Comment by Salamandar** [comment]
Yeah, no problem with that ! It's sometimes easier to review. I may squash them in the end.

**Pull request merged**